tests/test_bootstrap.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+from mozanalysis.bootstrap import run_bootstrap
+
+
+DATA = [1.0, 1.0, 1.0, 2.0, 2.0, 3.0, 8.0, 15.0]
+
+
+def test_mean():
+    assert np.mean(DATA) == 4.125
+
+
+def test_bootstrap_mean(spark_context):
+    d = run_bootstrap(spark_context, DATA, np.mean, num_iterations=10)
+    assert d['calculated_value'] == 4.125
+    assert d['confidence_low'] < d['calculated_value'] < d['confidence_high']
```

Comment:
Good start -- is the plan to add a test using a normal distribution as we discussed?

Reply:
Yes, I forgot about that. Thanks for the reminder.